### ISSUE TITLE
Access min-vid player from firefox menu

### ISSUE THREAD

**Issue by Azmisov** [meandavejustice/min-vid#1112]
Perhaps I simply cannot find it, but it appears min-vid can only be opened from the UI controls overlayed on enabled websites. Since min-vid keeps track of your old queue / history, it would be nice to be able to pull the player up from any site (e.g. using firefox menu), and resume your previous queue.

**Comment by simonwiles** [comment]
Is anybody else interested in this?  It feels really odd that to open my queue where I left off I have to go to a random video and either open it and then skip past it, or add it to the queue and then remove it again.  Would this be more difficult to implement than I'm assuming?

**Comment by SoftVision-PaulOiegas** [comment]
@meandavejustice @johngruen Maybe it would be an idea to add a toolbar button for Min Vid player launch? Seems a good idea now that we have queue/history.

**Comment by meandavejustice** [comment]
@simonwiles @SoftVision-PaulOiegas I agree the toolbar button especially makes sense when there are items in queue/history.

We are in a feature freeze at the moment with min vid until the shield study is run. Thanks for reporting @Azmisov, we'll definitely keep this in mind.
